crates/cli/src/subcommands/build.rs

```diff
@@ -7,10 +7,10 @@ pub fn cli() -> clap::Command {
     clap::Command::new("build")
         .about("Builds a spacetime module.")
         .arg(
-            Arg::new("project-path")
-                .default_value(".")
+            Arg::new("project_path")
                 .value_parser(clap::value_parser!(PathBuf))
-                .help("The path of the project that you would like to build."),
+                .default_value(".")
+                .help("The system path (absolute or relative) to the project you would like to build")
```

Comment:
tried to use more consistent text, arg ordering, and naming between subcommands where possible